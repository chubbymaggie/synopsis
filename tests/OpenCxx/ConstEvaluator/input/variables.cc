const int a = 1 + 2 - 3;
const int b = 1 << 4;
const int c = 8 | 4, d = 22;
int e;
int f = e;
const int g = 1 > 2 ? 1 : 2;
const int h = (24 >> 3);
const int i = (-1);
const int j = ~0;
const bool k = true;
const bool l = false;
