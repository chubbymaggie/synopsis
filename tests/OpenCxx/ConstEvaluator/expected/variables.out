initializer : 1 + 2 - 3
value : 0
initializer : 1 << 4
value : 16
initializer : 8 | 4
value : 12
initializer : 22
value : 22
initializer : e
value : none
initializer : 1 > 2 ? 1 : 2
value : 2
initializer : ( 24 >> 3 )
value : 3
initializer : ( - 1 )
value : -1
initializer : ~ 0
value : -1
